zjit/src/stats.rs

```diff
@@ -225,6 +239,26 @@ pub fn exit_counter_ptr(reason: crate::hir::SideExitReason) -> *mut u64 {
     counter_ptr(counter)
 }
 
+pub fn send_without_block_def_type_counter(def_type: crate::hir::SendWithoutBlockDefType) -> Counter {
```

Comment:
The counter prefix `send_fallback_` doesn't imply the `_without_block` part. Maybe just call it `send_fallback_counter` (or `send_fallback_method_type_counter` for now)?